src/script/lua_api/l_object.cpp

```diff
@@ -1716,32 +1716,25 @@ int ObjectRef::l_hud_get_hotbar_selected_image(lua_State *L)
 	return 1;
 }
 
-// set_sky(self, sky_parameters)
-int ObjectRef::l_set_sky(lua_State *L)
+void ObjectRef::set_sky(lua_State *L, int table_index, RemotePlayer *player)
 {
-	NO_MAP_LOCK_REQUIRED;
```

Comment:
Why does this suddenly require a map lock?

Reply:
it doesn't, these are stubs

they should be kept vaguely correct anyway